test/Microsoft.ML.Tests/TimeSeries.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Api;
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+
+namespace Microsoft.ML.Tests
+{
+    public sealed class TimeSeries
+    {
+
+        public class Prediction
+        {
+            [VectorType(4)]
+            public double[] Change;
+        }
+
+        sealed class Data
+        {
+            public float Value;
+
+            public Data(float value)
+            {
+                Value = value;
+            }
+        }
+
+        [Fact]
+        public void ChangeDetection()
+        {
+            using (var env = new ConsoleEnvironment(conc: 1))
+            {
+                const int size = 10;
+                List<Data> data = new List<Data>(size);
+                var dataView = env.CreateStreamingDataView(data);
+                List<Data> tempData = new List<Data>();
+                for (int i = 0; i < size / 2; i++)
+                    tempData.Add(new Data(5));
+
+                for (int i = 0; i < size / 2; i++)
+                    tempData.Add(new Data((float)(5 + i * 1.1)));
+
+                foreach (var d in tempData)
+                    data.Add(new Data(d.Value));
+
+                var args = new IidChangePointDetector.Arguments()
+                {
+                    Confidence = 80,
+                    Source = "Value",
+                    Name = "Change",
+                    ChangeHistoryLength = size,
+                    Data = dataView
+                };
+
+                var detector = TimeSeriesProcessing.IidChangePointDetector(env, args);
+                var output = detector.Model.Apply(env, dataView);
+                var enumerator = output.AsEnumerable<Prediction>(env, true).GetEnumerator();
+                Prediction row = null;
+                List<double> expectedValues = new List<double>() { 0, 5, 0.5, 5.1200000000000114E-08, 0, 5, 0.4999999995, 5.1200000046080209E-08, 0, 5, 0.4999999995, 5.1200000092160303E-08,
```

Comment:
>1200000000000114E [](start = 80, length = 17)

I think Assert.Equal has third parameter: precision, so you don't have, if you set it to 4, you don't have to have this long constants. (I'm actually surprised tests are not falling due to precision errors)